.pipelines/templates/publish-load-test-result.yaml

```diff
@@ -0,0 +1,19 @@
+steps:
+  - task: PublishTestResults@2
+    displayName: Publish Test Results
+    inputs:
+      testResultsFormat: "JUnit"
+      testResultsFiles: $(perf-tests.repo.path)/_artifacts/junit.xml
+    condition: succeededOrFailed()
+
+  - script: |
+      # rename : to -
+      for f in *:*; do mv -v "$f" $(echo "$f" | tr ':' '-'); done
+      mkdir -p $(Build.ArtifactStagingDirectory)/${POD_COUNT}
```

Comment:
where is `POD_COUNT` coming from? Since we are doing a single job, I don't this is needed here